src/test/FuzzerImpl.cpp

```diff
@@ -696,12 +696,17 @@ template <>
 void
 generator_t::operator()(stellar::PublicKey& t) const
 {
-    // note that we include NUMBER_OF_PREGENERATED_ACCOUNTS as to also cover the
-    // case of non existing accounts
+    // Generate account IDs in a somewhat larger range than the number of
```

Comment:
I think the point of the constraint was (and is) to make a larger fraction of the keys in the corpus correspond to existing accounts.  The fuzzer should figure out what's interesting, but it might figure out what's interesting faster if the interesting ones are denser within the corpus.  It might also be smart/fast enough that it doesn't make much difference, but in general the point of the compact XDR form has been to shrink the uninteresting parts of various spaces, so I think it's consistent to do so with accounts where the number of interesting ones is significantly smaller than fit in even one byte.

Reply:
Opened https://github.com/stellar/stellar-core/issues/2946.